backend/note_taking/migrations/0010_alter_block_options_and_more.py

```
@@ -0,0 +1,30 @@
+# Generated by Django 5.2.4 on 2025-09-02 12:35
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('note_taking', '0009_alter_block_options_and_more'),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)